libcontainer/specconv/spec_linux.go

```diff
@@ -219,8 +219,10 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 		config.Seccomp = seccomp
 	}
 	config.Sysctl = spec.Linux.Sysctl
-	if oomScoreAdj := spec.Linux.Resources.OOMScoreAdj; oomScoreAdj != nil {
-		config.OomScoreAdj = *oomScoreAdj
+	if spec.Linux.Resources != nil {
+		if oomScoreAdj := spec.Linux.Resources.OOMScoreAdj; oomScoreAdj != nil {
```

Comment:
Yeah, I'm confused why we did the if statement that way in the first place (I though it was calling a function ;]).